exercises/practice/bob/.approaches/switch-on-tuple/content.md

```diff
@@ -0,0 +1,47 @@
+# `switch` on a `tuple`
+
+```csharp
+    // uses System.Linq;
+    private static bool isShout(string input) => input.Any(c => char.IsLetter(c)) && input.ToUpper() == input;
```

Comment:
```suggestion
    private static bool IsShout(string input) => input.Any(c => char.IsLetter(c)) && input.ToUpper() == input;
```

And perhaps move this code after the `Response` method? I was usually quite strict about starting with `private` stuff first, but I no longer am. I now prefer reading from high-level to low-level, I also don't see a mention of the ordering of visibility in the [coding conventions](https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/coding-conventions). If you disagree, let me know.

Reply:
I'll modify the capitalization when I restore  the whole code.

Personally, in C# I prefer showing the definition of a local private variable or method before I use it,. It's not that I'm strict about it; it just feels right to me. If there is a lot of  private stuff, then I put it in a collapsible region (e.g. `#region private variables`), but I hardly ever uses regions in Exercism, since it's more of a VS thing than a language thing.

Reply:
I don't massively care either way, so feel free to keep it like this (except the capitalization).